environment.yml

```diff
@@ -17,32 +20,34 @@ dependencies:
 - nomkl=1.0
 - nose=1.3.7
 - numexpr=2.6.2
-- numpy=1.12.1
+- numpy=1.13.1
 - openblas=0.2.19
-- pandas=0.20.1
+- pandas=0.20.3
 - patsy=0.4.1
 - psycopg2=2.6.2
+- pyarrow=0.5.0
 - pycrypto=2.6.1
-- pytest=3.0.5
-- python=3.6.1
+- pytest=3.1.3
+- python=3.6.2
 - pyyaml=3.12
 - requests=2.14.2
-- seaborn=0.7.1
-- scipy=0.19.0
-- scikit-learn=0.18.1
+- seaborn=0.8
+- scipy=0.19.1
+- scikit-learn=0.18.2
 - statsmodels=0.8.0
 - pip:
   - awscli==1.11.75
-  - civis==1.5.2
+  - civis==1.6.0
+  - cloudpickle==0.3.1
   - dropbox==7.1.1
   - ftputil==3.3.1
   - glmnet==2.0.0
   - joblib==0.11.0
   - muffnn==1.1.2
-  - pubnub==4.0.10
+  - pubnub==4.0.12
   - pysftp==0.2.9
   - python-simple-hipchat==0.4.0
-  - requests-toolbelt==0.7.1
-  - tensorflow==1.1.0
+  - requests-toolbelt==0.8.0
+  - tensorflow==1.2.1
   - urllib3==1.19
   - xgboost==0.6a2
```

Comment:
In fact, we could probably be getting a lot of this stuff from conda forge if we want to go all in on it.